### PR TITLE
Update get_status signature

### PR DESCRIPTION
**Pull request by npentrel**
not quite sure how to fix the errors

### CODE REVIEW

File: src/viam/robot/client.py [hunk 1]
```diff
@@ -599,7 +599,7 @@ async def __aexit__(self, exc_type, exc_value, traceback):
     ##########
     # STATUS #
     ##########
-    async def get_status(self, components: Optional[List[ResourceName]] = None):
+    async def get_status(self, components: Optional[List[ResourceName]] = None) -> List[Status]:
```
Comment (npentrel) [inline src/viam/robot/client.py:602]:
```suggestion
    async def get_status(self, components: Optional[List[ResourceName]] = None) -> List[viam.proto.robot.Status]:
```

File: src/viam/robot/client.py [hunk 2]
```diff
@@ -599,7 +599,7 @@ async def __aexit__(self, exc_type, exc_value, traceback):
     ##########
     # STATUS #
     ##########
-    async def get_status(self, components: Optional[List[ResourceName]] = None):
+    async def get_status(self, components: Optional[List[ResourceName]] = None) -> List[viam.proto.robot.Status]:
```
Comment (npentrel) [inline src/viam/robot/client.py:602]:
```suggestion
    async def get_status(self, components: Optional[List[ResourceName]] = None) -> List[robot.Status]:
```

File: src/viam/robot/client.py [hunk 3]
```diff
@@ -599,7 +600,7 @@ async def __aexit__(self, exc_type, exc_value, traceback):
     ##########
     # STATUS #
     ##########
-    async def get_status(self, components: Optional[List[ResourceName]] = None):
+    async def get_status(self, components: Optional[List[ResourceName]] = None) -> List[robot.Status]:
```
Comment (npentrel) [inline src/viam/robot/client.py:603]:
```suggestion
    async def get_status(self, components: Optional[List[ResourceName]] = None) -> List[Status]:
```

**Comment by njooma** [comment]
I fixed the issue -- there was a naming collision when you imported `Status` -- we were already importing another field named `Status`

**Review by njooma — APPROVED** [review]

**Pull request merged**